tests/test_nc_to_mmd.py

```diff
@@ -106,10 +94,10 @@ def test_get_related_dataset(dataDir):
 def test_get_related_dataset_OLD(dataDir):
     """ Test get_related_dataset_OLD function.
     """
-    # Relation is correctly defined according to old standard
     mmd_yaml = yaml.load(
```

Comment:
This one too

Reply:
this is just done once in the function. Should be ok?